### ISSUE TITLE
The second call of magento:setup:permissions is skipped

### ISSUE THREAD

**Issue by erikhansen** [davidalger/capistrano-magento2#18]
In Capistrano 3.6, tasks can't be run twice without explicitly setting "reenable" (see https://github.com/capistrano/capistrano/issues/1686).

![10-57-57 cap -bash 156x69-5gmmg](https://cloud.githubusercontent.com/assets/129031/18011734/fb482d9c-6b7b-11e6-80ae-c30652b791e4.png)

While I haven't experienced any problems caused by the second call to `magento:setup:permissions` being skipped, the behavior of this Gem needs to be changed to take into consideration the Capistrano 3.6 behavior. So either explicitly set "reenable" or change `magento:setup:permissions` to only be called once.

**Comment by davidalger** [comment]
At first read I took what you wrote as implying that 3.6 changed something which caused the issue, but this is in-fact just a discussion where changing capistrano to diverge from default rake DSL behavior resulted in adding a warning to inform users who's procedures attempted to rely on an incorrectly assumed behavior (such as this one does).

Apparently I never noticed in the output that the permissions reset was never run the 2nd time, and 3.6+ now throws a warning. So…sounds like a bug fix is in order. I've confirmed that it doesn't run twice currently in the older version to prove my reading of the issue thread correct ;)

Now what I'm trying to figure out…

a) is it necessary to apply the permissions reset after generation of code / static assets

b) can it be run once, but after the generation is complete vs before and after

The advantage of doing it after is clear: everything has proper permissions. Running it before may be strictly necessary in _some_ edge cases. The goal of this command is to have it lock down permissions to only be as open as necessary for M2 to run in most environments. That would include (most likely) read only permissions for everyone but the owner on things like generated static assets and code files, as well as the vendor directory.

**Comment by erikhansen** [comment]
@davidalger Yeah, you are right that the behavior wasn't _introduced_ in 3.6, but rather a notification was added explaining the behavior. Thanks for catching that.

I know there was a reason my initial Cap scripts reset the permissions twice, but I had a somewhat unique Linux user setup that I've since moved away from. I tend to think option b should work without issues in a standard setup. If we drop the first permissions reset and it causes issues, it should cause an atomic failure of the deployment, so the risk of making this change is fairly low.

If you'd like, I can remove the first permissions reset locally and test for a while and then submit it as a PR if I don't see any issues.
